api/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	configv1alpha1 "k8s.io/component-base/config/v1alpha1"
+	"k8s.io/utils/ptr"
+)
+
+const (
+	DefaultWebhookCertDir                      = "/tmp/k8s-webhook-server/serving-certs"
+	DefaultWebhookServiceName                  = "lws-webhook-service"
+	DefaultWebhookSecretName                   = "lws-webhook-server-cert"
+	DefaultWebhookPort                         = 9443
+	DefaultHealthProbeBindAddress              = ":8081"
+	DefaultReadinessEndpoint                   = "/readyz"
+	DefaultLivenessEndpoint                    = "/healthz"
+	DefaultMetricsBindAddress                  = ":8443"
+	DefaultLeaderElectionID                    = "b8b2488c.x-k8s.io"
+	DefaultLeaderElectionLeaseDuration         = 15 * time.Second
+	DefaultLeaderElectionRenewDeadline         = 10 * time.Second
+	DefaultLeaderElectionRetryPeriod           = 2 * time.Second
```

Comment:
I would move those to the test file since changing them is not going to actually change the default.